### DISCUSSION TITLE
Please allow access to your mod for third party lanuchers.

### DISCUSSION THREAD

**Discussion by blackd** [IntelligenceModding/AdvancedPeripherals#303] [General]
With the new curseforge REST API rollout your mods are currently only accessible using the curseforge site and the curseforge app.

Could you please allow access for launchers like ATLauncher.

For more info https://support.curseforge.com/en/support/solutions/articles/9000207877-project-distribution-toggle

**Comment by SirEndii** [comment]
After some discussions and a bit of thinking, I enabled third party distribution
You can now use it with third party launchers (:
